src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/CmdLineClient.kt

```diff
@@ -100,14 +101,14 @@ class ResultReceiver(
         private val expectedTaskId: String
 ) : CompilerFleetMessageReceiver() {
     override fun processMessage(message: PubsubMessage): Boolean {
-        val result = CompilerFleet.CompilerFleetResult.parseFrom(message.data)
+        val result = CompilerFleet.CompilerFleetResult.parseFrom(message.data).identifier
 
         if (result.taskId != expectedTaskId) {
             LOGGER.info("Task ids don't match: \nexpected:{}, \nactual:{}", expectedTaskId, result.taskId)
             return false
         }
 
-        FileUtils.writeByteArrayToFile(outputFile, result.resultBytes.toByteArray())
+        FileUtils.writeByteArrayToFile(outputFile, result.taskId.toByteArray())
```

Comment:
A little bit strange to write task id instead of PDF bytes. Especially given that you don't seem to set identifier as one of those oneof's in `CompilerFleetResult` anywhere in the code.

Reply:
Fixed